src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -592,7 +592,10 @@ Empty-room MEG recordings capture the environmental and recording system's
 noise.
 In the context of BIDS it is RECOMMENDED to perform an empty-room recording for
 each experimental session.
-It is RECOMMENDED to store the empty-room recording inside a subject directory
+
+If empty-room recordings are not done prior to each individual experimental session, but
+on a daily schedule, one empty-room recording may have to be used with multiple subjects and/or sessions. In that case
+it is RECOMMENDED to store the empty-room recording inside a subject directory
```

Comment:
Store it inside … which subject's directory? The `sub-emptyroom`? or `sub-participant`? I find this now confusing in combination with the two examples provided.

Reply:
Ooooh!! I actually missed these last two words! Sorry about the confusion!!